policy/modules/services/zfs.te

```diff
@@ -112,9 +122,35 @@ mount_exec(zfs_t)
 
 userdom_use_user_terminals(zfs_t)
 
+zfs_rw_zpool_cache(zfs_t)
+
 optional_policy(`
 	kernel_rw_rpc_sysctls(zfs_t)
 
 	rpc_manage_nfs_state_data(zfs_t)
 	rpc_read_exports(zfs_t)
 ')
+
+#######################################
+#
+# Mail local policy
+#
+
+optional_policy(`
+	mta_base_mail_template(zed)
+	role system_r types zed_mail_t;
+
+	allow zed_mail_t zed_t:fd use;
+	allow zed_mail_t zed_t:fifo_file rw_fifo_file_perms;
+	allow zed_mail_t zed_t:process sigchld;
+
+	manage_dirs_pattern(zed_t, zed_mail_tmp_t, zed_mail_tmp_t)
+	manage_files_pattern(zed_t, zed_mail_tmp_t, zed_mail_tmp_t)
+	files_tmp_filetrans(zed_t, zed_mail_tmp_t, { dir file })
+
+	mta_sendmail_domtrans(zed_t, zed_mail_t)
+
+	allow zed_mail_t self:capability { dac_override dac_read_search };
+
+	storage_raw_rw_fixed_disk(zed_mail_t)
```

Comment:
This is very concerning, a networked process with raw disk access.

Reply:
I'll see if there's a workaround or some other way we can avoid this.

Reply:
If not, please make a tunable. Then it will at least make the users aware of the access.

Reply:
That was my next question. :)

Reply:
I found that this access is not needed at all yet for some reason is attempted by the `mail` process that gets spawned. I ended up just `dontaudit`ing it.

Reply:
Perhaps a leaked fd?